### ISSUE TITLE
Undefined index: ccn in PHPLOC

### ISSUE THREAD

**Issue by hkdobrev** [Qafoo/QualityAnalyzer#230]
Hi,

I'm getting the following warning:
```
PHP Notice:  Undefined index: ccn in /Users/hkdobrev/projects/QualityAnalyzer/vendor/phploc/phploc/src/Analyser.php on line 507
```

<details>
<summary>Click to view full log</summary>
$ ~/projects/QualityAnalyzer/bin/analyze analyze -c build/coverage-clover.xml -t build/tests-junit.xml -vvv src/
Analyze source code in ~/projects/clippings/src
 * Running source
 * Running coverage
 * Running pdepend
 * Running dependencies
 * Running phpmd
 * Running checkstyle
 * Running tests
 * Running cpd
 * Running phploc
PHP Notice:  Undefined index: ccn in ~/projects/QualityAnalyzer/vendor/phploc/phploc/src/Analyser.php on line 507
PHP Notice:  Undefined index: ccn in ~/projects/QualityAnalyzer/vendor/phploc/phploc/src/Analyser.php on line 507
PHP Notice:  Undefined index: ccn in ~/projects/QualityAnalyzer/vendor/phploc/phploc/src/Analyser.php on line 507
PHP Notice:  Undefined index: ccn in ~/projects/QualityAnalyzer/vendor/phploc/phploc/src/Analyser.php on line 507
PHP Notice:  Undefined index: ccn in ~/projects/QualityAnalyzer/vendor/phploc/phploc/src/Analyser.php on line 507
 * Running git
 * Running gitDetailed  <br>
.  <br>
.  <br>
.
<hr>
</details>  
<br>

However, when I run PHP LOC independently, I don't get this warning with the same codebase.

Let me know if I could be of any further help investigating this.

**Comment by kore** [comment]
It seems to be this issue – I get it too in some projects: https://github.com/sebastianbergmann/phploc/issues/155 - looks like we must wait for a new release of PHPLOC.
